src/pages/lessons/[slug].tsx

```diff
@@ -217,7 +217,7 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
         'loaded',
         'viewing',
         'completed',
-        'autoPlaying',
+        'showingNext',
```

Comment:
I'm trying to "churn" the player less and reuse the instance instead of hiding it and bringing it back

still not "just right" but better